src/azure-cli/azure/cli/command_modules/eventhubs/Operation/Namespace_custom_file.py

```diff
@@ -0,0 +1,227 @@
+                         capacity=None,  tier='Standard', mi_user_assigned=None, mi_system_assigned=False,
+                          encryption_config=None, minimum_tls_version=None,disable_local_auth = None,
+                         maximum_throughput_units=None,private_endpoint_connection_name='enabled',
+                         require_infrastructure_encryption=None,is_kafka_enabled=None,
+                              is_auto_inflate_enabled=None,alternate_name=None, zone_redundant=None):
+    from azure.cli.command_modules.eventhubs.aaz.latest.eventhubs.namespace import Create
+    user_assign = {}
+    dict1 = {}
+    a = "None"
+    print(encryption_config)
+    dict1 = {
+        "resource_group": resource_group_name,
+        "namespace_name": namespace_name,
+        "tags": tags,
+        "sku": {
+            "name": sku,
+            "capacity": capacity,
+            "tier": sku
+        },
+        "minimum_tls_version": minimum_tls_version,
+        "location": location,
+        "enable_auto_inflate": is_auto_inflate_enabled,
+        "maximum_throughput_units":maximum_throughput_units,
+        "is_kafka_enabled":is_kafka_enabled,
+        "zone_redundant": zone_redundant,
+        "private_endpoint_connection_name": private_endpoint_connection_name,
+        "disable_local_auth": disable_local_auth,
+        "alternate_name": alternate_name
+    }
+    if mi_system_assigned:
+        a = "SystemAssigned"
+    if mi_user_assigned:
+        if mi_system_assigned:
+            a = "SystemAssigned, UserAssigned"
+        else:
+            a = "UserAssigned"
+
+        for col in mi_user_assigned:
+            user_assign[col] = {}
+        dict2 = {
+            "identity": {
+                "type": a,
+                "user_assigned_identities": user_assign
+            },
+        }
+        dict1.update(dict2)
+    else:
+        dict2 = {
+            "identity": {
+                "type": a,
+                "user_assigned_identities": None
+            },
+        }
+        dict1.update(dict2)
+    if encryption_config:
+        dict3 = {
+            "encryption": {
+                "key_vault_properties": encryption_config,
+                "key_source": "Microsoft.KeyVault",
+                "require_infrastructure_encryption":require_infrastructure_encryption
+            }
+        }
+        dict1.update(dict3)
+    print(dict1)
+    return Create(cli_ctx=cmd.cli_ctx)(command_args=dict1)
+
+def cli_add_encryption(cmd, resource_group_name, namespace_name, encryption_config,require_infrastructure_encryption=None):
+    from azure.cli.command_modules.eventhubs.aaz.latest.eventhubs.namespace import Update
+    from azure.cli.command_modules.eventhubs.aaz.latest.eventhubs.namespace import Create
+    from azure.cli.command_modules.eventhubs.aaz.latest.eventhubs.namespace import Show
+    eventhubsnm = Show(cli_ctx=cmd.cli_ctx)(command_args={
```

Comment:
provide for new lines after all from statements and wherever necessary in other places